vms/platformvm/blocks/builder/builder.go

```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2022, Chain4Travel AG. All rights reserved.
```

Comment:
I suggest fo make a camino-builder, it is (except tests) only instanciated in vm.go